backend/pycon/settings/base.py

```diff
@@ -195,7 +195,12 @@
     "disable_existing_loggers": False,
     "handlers": {"console": {"class": "logging.StreamHandler"}},
     "loggers": {
-        "app_api": {"handlers": ["console"], "level": "WARNING"},
+        "pycon.api": {"handlers": ["console"], "level": "WARNING", "propagate": True},
+        "pycon.integrations": {
+            "handlers": ["console"],
+            "level": "WARNING",
+            "propagate": True,
+        },
         "qinspect": {"handlers": ["console"], "level": "DEBUG", "propagate": True},
```

Comment:
does this work the same?

```suggestion
        "pycon": {"handlers": ["console"], "level": "WARNING", "propagate": 
        "qinspect": {"handlers": ["console"], "level": "DEBUG", "propagate": True},
```

Reply:
think before we were missing some logs, `app_api` was wrong. 
If you look at [Django logging docs](https://docs.djangoproject.com/en/3.2/topics/logging/#configuring-logging) the correct configuration is ` 'myproject.custom'`. 
And also `propagate` means you configure the log to make it work also in the submodules.